### PR TITLE
Add metadata output of author order notebook in analysis repo

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Here, I'm adding the YAML output of [`analyses/count-contributions/03-set-authorship-order.Rmd`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/82966bc74343a641975f19ab5790567dc49e7bdb/analyses/count-contributions/03-set-authorship-order.Rmd) in the `AlexsLemonade/OpenPBTA-analysis` repository. The git contributions information that is currently in the `master` branch was updated via the cron job 1 week ago, but I don't really expect that to change the underlying order of the authors when the cron job runs in about an hour.

There are a bunch of whitespace changes in the metadata. (I'd probably recommend hiding whitespace changes in the interface when someone reviews eventually.) I'm adding the unaltered output of `yaml::write_yaml()`. So for now, I'm going to mark this as a draft and make sure the YAML is still valid from Manubot's point of view.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
This built just fine, so I am going to mark ready for review and request reviewers.

**Review by jharenza — COMMENTED** [review]
This generally looks good, except that I realized Yang Yang was missing from the analysis module counts. Updated in [this PR](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1600). Other module contributors look good and in the proper order.

File: content/metadata.yaml [hunk 1]
```diff
@@ -1,680 +1,656 @@
...
+  - Validation
+  - Formal analysis
+  - Writing - Review and editing
+  - Visualization
+  - Supervision
+  affiliations:
+  - Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA,
+    USA<sup>☯</sup>
+  - Rowan University, Glassboro, NJ, USA
+  symbol_str: ☯
```
Comment (jashapiro) [inline content/metadata.yaml:77]: I just want to note that this symbol renders as an emoji ☯️, which we probably don't want. Substitute † or ‡?

**Review by jashapiro — APPROVED** [review]
LGTM. I made a comment about a symbol change, but it doesn't have to happen here.

**Review by jharenza — APPROVED** [review]
Going to approve given that https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1600 was merged. This would not make a subtantial change to order since Yang Yang is toward the top of the randomly-selected author list, so I think we can distribute to authors with this order.

**Pull request merged**